src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -1626,7 +1626,7 @@ private static void ReadTargetFrameworks(PackageSpec packageSpec, JsonTextReader
             NuGetFramework frameworkName = NuGetFramework.Parse((string)jsonReader.Value);
 
             var targetFrameworkInformation = new TargetFrameworkInformation();
-
+            NuGetFramework secondaryFramework = default;
```

Comment:
fancy way to write `null`? 😝

Reply:
Interchangeable :) 

I've switched to using default because of generics.